src/distfile.py

```diff
@@ -63,7 +65,7 @@ def _apply_patch(patch, target):
                     break
 
         # Apply the patch.
-        with open(patch) as f:
+        with open(patch, 'rb') as f:
```

Comment:
Is this change trying to fix some specific bug/regression?

Reply:
I think so, but I rushed the commit messages and of course now I don't remember clearly.

Perhaps it's not actually necessary... perhaps it's left over from flailing around. When a variable like f gets used as more than one type in the same function, mypy's diagnostics are bad and I was pulling my hair out trying to figure out what was going on.

Reply:
Ah... right... earlier in that function, line 47, we have `with open(patch, 'rb') as f:` so f has type `BinaryIO`. `open(patch)` returns `TextIO`. Without this `rb` arg, we get:

```
src/distfile.py:68: error: Incompatible types in assignment (expression has type "TextIO", variable has type "BinaryIO")
```

Reply:
Sounds all right then!